tracee/tracee.go

```diff
@@ -429,9 +435,39 @@ func (t *Tracee) populateBPFMaps() error {
 		fileFilterMap.Update(uint32(i), []byte(t.config.FilterFileWrite[i]))
 	}
 
-	uidHash, _ := t.bpfModule.GetMap("uid_filter")
-	for i := 0; i < len(t.config.Filter.UIDs); i++ {
-		uidHash.Update(uint32(t.config.Filter.UIDs[i]), uint32(1))
+	// Set filter options
+	if t.config.Filter != nil {
```

Comment:
If no filter chosen, no events will show as uidCompareFilter defaults are not set

Reply:
Right - didn't get that far to test because I'm in verifier hell, are you have the same issue if you've updated your kernel?

Reply:
Verifier hell! I know the feeling....

See my suggestion in the other comment

Reply:
I'm really looking forward to one day understanding the nuances of the verifier, just wish I can figure out this particular issue so I can at least learn from it :upside_down_face: